ci-operator/config/quay/quay-tests/quay-quay-tests-master__quay-conformance.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: quay-conformance-test-quay311
+  cron: 0 12 1 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m5.8xlarge
+      ODF_OPERATOR_CHANNEL: stable-4.15
+      QUAY_INDEX_IMAGE_BUILD: brew.registry.redhat.io/rh-osbs/iib:713080
+      QUAY_OPERATOR_CHANNEL: stable-3.11
+    test:
+    - ref: quay-tests-deploy-quay-odf
+    - ref: quay-tests-quay-conformance-test
+    workflow: cucushift-installer-rehearse-aws-ipi
+zz_generated_metadata:
+  branch: master
+  org: quay
+  repo: quay-tests
+  variant: quay-conformance
```

Comment:
suggest use name "quay-oci-conformance" to show that's **OCI Conformance Testing**